code/src/components/messageList.css

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
Similar to the CSS for NewThought, I wonder if the .message-container and .message-box classes are competing? Also here, maybe try a combination of width with view width (vw) as the unit of measurement and max-width to control for desktop sizing.